tests/sentry/api/endpoints/test_organization_profiling_profiles.py

```diff
@@ -810,6 +811,7 @@ def test_queries_profile_candidates_from_profiles_with_continuous_profiles_with_
 
     @patch("sentry.profiles.flamegraph.bulk_snuba_queries")
     @patch("sentry.api.endpoints.organization_profiling_profiles.proxy_profiling_service")
+    @pytest.mark.skip(reason="Intermittent failure")
     def test_queries_profile_candidates_from_spans(
```

Comment:
iirc the mark decorator has to be the outermost one to work?  

can you link an issue here ala https://github.com/getsentry/rfcs/blob/main/text/0027-manual-disabling-of-flaky-tests.md